sqswatcher/sqswatcher.py

```diff
@@ -139,7 +144,7 @@ def _setup_ddb_table(region, table_name, proxy_config):
 def _retry_on_request_limit_exceeded(func):
     @retry(
         stop_max_attempt_number=3,
-        wait_exponential_multiplier=15000,
+        wait_exponential_multiplier=5000,
```

Comment:
We could increase the max_attempt_number since we are decreasing the multiplier.

Reply:
increased to 5